events-application-rest-api/src/main/java/com/bounswe/bounswe2017group3/UserRepository.java

```diff
@@ -1,8 +1,21 @@
 package com.bounswe.bounswe2017group3;
 
+import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface UserRepository extends JpaRepository<User, Long> {
+        
+    public User findByUsername(String username);
+
+    public User findById(long id);
+
+    @Query("SELECT COUNT(u) FROM User u where u.username = :username") 
```

Comment:
I think it can be simplified to: `SELECT COUNT(*) FROM user WHERE username = :username`. Just a feedback, it's OK this way too 😄